### ISSUE TITLE
Mounted Dragon's Breath Balance Tweak

### ISSUE THREAD

**Issue by Spinogaming** [AlexModGuy/Ice-and-Fire-Suggestions#824]
When mounting a dragon, there could be a charge for when you can use the breath attack. It would take the place of the exp bar and look like it but red or orange for fire, and bluish white for ice. You can use the breath attack all the way until it's empty. It would slowly recharge until full. Higher stages have a longer lasting breath meter

**Comment by RainbowTurtle-skylord5201** [comment]
I think this is already a thing, just not shown visually (in the bestiary it says that older dragons have a longer breath or something). Having the bar would be nice, though!

**Comment by Spinogaming** [comment]
Nope, if you hold down the breath biutton they continously breathe
